packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx

```diff
@@ -84,7 +84,7 @@ const DefaultPaymentLegalBlurb = () => (
         ),
         paypalPrivacyLink: (
           <a
-            href="https://paypal.com/privacy"
+            href="https://www.paypal.com/webapps/mpp/ua/privacy-full"
```

Comment:
Looks like https://paypal.com/privacy redirects to ^. Maybe it is better to keep link as is?

Reply:
> 
> 
> Looks like https://paypal.com/privacy redirects to ^. Maybe it is better to keep link as is?

Seconded.  https://paypal.com/privacy looks more like a permalink.

Reply:
It did one one day and didn't on another day more recently. This is the link [Tony asked us to use](https://mozilla-hub.atlassian.net/browse/FXA-3715?focusedCommentId=441990) in the ticket.

Reply:
> 
> 
> It did one one day and didn't on another day more recently

LOL why is PayPal so terrible :'(